scripts/bootstrap.js

```diff
@@ -68,6 +62,20 @@ function run() {
       pre: ['install', 'build', 'manager'],
       order: 1,
     }),
+    retry: createTask({
+      name: `Core & Examples but only build previously failed ${chalk.gray('(core)')}`,
+      defaultValue: true,
+      option: '--retry',
+      command: () => {
+        log.info(prefix, 'prepare');
+        spawn(
+          `nx run-many --target=prepare --all --parallel --only-failed ${
+            process.env.CI ? `--max-parallel=${maxConcurrentTasks}` : ''
+          }`
+        );
+      },
+      order: 1,
+    }),
```

Comment:
@tmeasday asked for this.

When `nx` fails it tells you you should re-run with a `--only-failed` flag.
I implemented this flag into the bootstrap command.

Just do this:

```sh
yarn bootstrap --retry
```